README.md

```diff
@@ -41,6 +40,42 @@ and whilst rails is an excellent tool, for smaller projects you may want to look
 as sinatra etc...
 
 
+### Client IP address tracing in Rails apps
+
+
+Add the request store gem to your Gemfile and install
+
+```ruby
+gem "request-store"
```

Comment:
We could add this to the gemspec

Reply:
I'm in two minds as it's not a requirement to use `RequestStore`, but also I can't think of another way to make use of the config lambda 🤷‍♂